apps/class-solid/src/components/Analysis.tsx

```diff
@@ -199,11 +225,12 @@ export function VerticalProfilePlot({
     <>
       <div class="flex flex-col gap-2">
         <ChartContainer>
-          <Legend entries={profileData} />
+          <Legend entries={() => [...profileData(), ...observations()]} />
           <Chart title="Vertical profile plot">
             <AxisBottom domain={xLim} label={analysis.variable} />
             <AxisLeft domain={yLim} label="Height[m]" />
             <For each={profileData()}>{(d) => Line(d)}</For>
+            <For each={observations()}>{(d) => Line(d)}</For>
```

Comment:
Why show as a dotted line? 

This gives the impression that each dot is an observation event. 

Also rendering as dotted line makes it look like second experiment (which is also a dotted line)

Reply:
Can do this as part of https://github.com/classmodel/class-web/issues/110#issuecomment-2785557586